files/ldap-stg/ldifs/README.md

```diff
@@ -1,9 +1,7 @@
 # Bootstrap data
 
 The ldif files will create all the basic structure of the ldap: <https://wiki.openmrs.org/x/6QDfAw>
-  - Enable ppolicy module
   - Create all `ou=policy` nodes
   - Create top nodes `ou=users`, `ou=groups`, `ou=system`
   - Creates all system users
   - Grant permissions to system users
-  - Change olcsizelimit <https://issues.openmrs.org/browse/ITSM-4251> to allow more than 500 users to be synced in crowd. Pagination didn't work for some reason.
```

Comment:
I removed the previous `bootstrap` folder and moved the readme here. 

So, the only files that are not on the full diff are `90-permissions.ldif`, `91-oldcsizelimit.ldif` and `95-ppolicy.ldif`

I have a feeling neither 91 and 95 are needed anymore, but... what about 90? Do we need to also add it to the new full diff? I'm not sure what it does honestly

Reply:
I'm guessing we might need 90. We can always test it without these permissions and add it later if it doesn't work as intended